### PR TITLE
Fixes #2496 fetal vein EC definition

### LINKED ISSUE #2496: [NTR] Fetal venous EC

### PR DESCRIPTION
**Pull request by JABelfiore**
fetal vein EC definition, synonyms, refs added

### CODE REVIEW

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
Can you please give the PR a more informative title

**Comment by anitacaron** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32333,6 +32334,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34497389") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4047006 "fetal artery endothelial cell")
 SubClassOf(obo:CL_4047006 obo:CL_1000413)
 
+# Class: obo:CL_4047011 (fetal vein endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1101/2021.04.07.438755") obo:IAO_0000115 obo:CL_4047011 "An endothelial cell that lines the veins in the fetal circulatory system.")
+AnnotationAssertion(terms:contributor obo:CL_4047011 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047011 "2024-09-05T15:45:32Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1101/2021.04.07.438755") oboInOwl:hasExactSynonym obo:CL_4047011 "fetal vein EC")
```
Comment (Caroline-99) [inline src/ontology/cl-edit.owl:32342]: please change to related synonyms and has_synonym_type : abbreviations

**Review by Caroline-99 — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (JABelfiore) [reply src/ontology/cl-edit.owl:32342]: amended now

**Review by Caroline-99 — APPROVED** [review]

**Pull request merged**